.coderabbit.yaml

```
@@ -0,0 +1,169 @@
+    project_keys: []
+  linear:
+    usage: auto
+    team_keys: []
+  pull_requests:
+    scope: auto
+code_generation:
+  docstrings:
+    language: en-US
+    path_instructions: []
+  unit_tests:
+    path_instructions: []
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)